src/main/scala/diplomacy/Nodes.scala

```diff
@@ -1235,14 +1235,14 @@ sealed abstract class MixedNode[DI, UI, EI, BI <: Data, DO, UO, EO, BO <: Data](
       data   = bundleIn(i))
   }
 
-  private var bundlesSafeNow = false
+  private[diplomacy] var instantiated = false
 
   /** Gather Bundle and edge parameters of outward ports.
     *
     * Accessors to the result of negotiation to be used within [[LazyModuleImp]] Code.
     */
   def out: Seq[(BO, EO)] = {
-    require(bundlesSafeNow, s"$name.out should only be called from the context of ${scope.get.name}'s LazyModuleImp, but the current scope is ${LazyModule.scope}.")
+    require(instantiated, s"$name.out should only be called after its parent LazyModule.module has been instantiated")
```

Comment:
Feel like text this isn't quite correct, since you can still use the thing during the execution of the `LazyModuleImp`, although I guess you would never trip this requirement at that time. Maybe `s"$name.out should not be called until after instantiation of its parent LazyModule.module has begun"`?